packages/iles/src/node/build/utils.ts

```diff
@@ -40,7 +40,8 @@ export function flattenPath (path: string) {
 // Paths ending with '/' are represented with index.html files.
 export function pathToFilename (path: string, ext = '') {
   if (path.endsWith(ext)) ext = ''
-  return `${(path.endsWith('/') ? `${path}index` : path).replace(/^\//g, '')}${ext}`
+  const decodedPath = path.split('%2F').map(decodeURIComponent).join('%2F')
+  return `${(decodedPath.endsWith('/') ? `${decodedPath}index` : decodedPath).replace(/^\//g, '')}${ext}`
```

Comment:
I'm not sure about slashes being encoded (`%2F`), as it would mean this check never adds `index` as the filename (which it does).

`vue-router` probably doesn't encode slashes, it should only be encoding certain path segments.

Reply:
I guess the slashes as delimiters are not encoded, but slashes as path params are encoded? Not very sure though, maybe some tests are needed.

Reply:
That makes sense, so I think we should do `split('/').map(decodeURIComponent).join('/')`, which should handle slashes in the dynamic paths as well.

Reply:
After some testing, now I think it's just fine to do `decodeURIComponent(path)` without special handling of slashes. But isn't `split('/').map(decodeURIComponent).join('/')` the same as `decodeURIComponent`?